moon/vehicles/rover.py

```diff
@@ -198,7 +193,8 @@ def get_steering_and_effort(self):
 
         elif self.drive_radius == 0:
             # turning in place if radius is 0 but speed is non-zero
-            e = 30
+            e = 30 * abs(self.drive_speed) / 1000.0 # 30 is max for turning in place
```

Comment:
`` a fixed effort for in-place turn is probably sufficient for now
```suggestion
            e = 30
```